### PR TITLE
Convert input checkbox to class

### PR DESCRIPTION
**Pull request by lovisaNamely**
Breaking up the form file.

Adding two classes `.checkbox-white` and `.checkbox-blue`, deprecating `[type="checkbox"]` selector.

### CODE REVIEW

**Comment by lovisaNamely** [comment]
@tsargent how do I add reviewers??? also. these changes shouldn't be breaking.

File: scss/elements/forms/_checkbox.scss [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  &:disabled:checked::after {
+    border: 0;
+    cursor: default;
+    margin-left: 1px;
+    margin-top: 1px;
+  }
+}
+
+/* ------- WHITE CHECKBOX, GREY TICK ------- */
+.checkbox-white[type=checkbox] {
```
Comment (johnholdun) [inline scss/elements/forms/_checkbox.scss:72]: the `[type=checkbox]` part of this can also be removed when the deprecated styles are removed, right?

Comment (lovisaNamely) [reply scss/elements/forms/_checkbox.scss:72]: yes! absolutely, its an annoying _three-step-deprecation_ thing going on. I am not sure this is feasible for ALL styles...

**Comment by johnholdun** [comment]
I like this strategy 👍

**Review by johnholdun — APPROVED** [review]

File: scss/elements/forms/_input.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
+@import 'checkbox';
```
Comment (HipsterBrown) [inline scss/elements/forms/_input.scss]: Why is `checkbox` imported here and in the `_forms.scss` file?

Comment (lovisaNamely) [reply scss/elements/forms/_input.scss]: that's wrong... hm which file should we pick? forms maybe?

**Review by HipsterBrown — APPROVED** [review]

**Comment by lovisaNamely** [comment]
coool! @tsargent @HipsterBrown what's the next step for making a new version of this? I suspect this is a minor

**Pull request merged**